### PR TITLE
Remove outputs from base workspace

### PR DESCRIPTION
**Pull request by ross-p-smith**
## What is being addressed
There is a bug in the porter installation of the base workspace stating that outputs are not present.

## How is this addressed

- remove outputs for now

### CODE REVIEW

File: templates/core/terraform/.terraform.lock.hcl [hunk 1]
```diff
@@ -58,21 +58,21 @@ provider "registry.terraform.io/hashicorp/local" {
 }
 
 provider "registry.terraform.io/hashicorp/random" {
-  version = "3.1.2"
```
Comment (tanya-borisova) [inline templates/core/terraform/.terraform.lock.hcl:61]: Does this need to change?

**Review by tanya-borisova — APPROVED** [review]
Just one question

File: templates/core/terraform/.terraform.lock.hcl (hunk 1 above)
Comment (ross-p-smith) [reply templates/core/terraform/.terraform.lock.hcl]: Removed - must have been updated by mistake - thanks! :-)

**Pull request merged**